.github/ISSUE_TEMPLATE/feature-vorschlag.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature Vorschlag
+about: Schlage ein neues Feature für DDP vor
+title: "[Feature Vorschlag] <Titel>"
```

Comment:
```suggestion
title: "Schreibe einen Titel"
```
Ich finde wir brauchen kein Prefix für issue Titel. Dafür sind die labels.

Reply:
das "[Fieature Vorschlag]" war zur einfachen übersicht in den Issues gedacht.
Dann brauchen wir stattdessen eine gut sichtbare, unique Farbe für das label